docs/api/v1/resourceset.md

```diff
@@ -152,6 +152,25 @@ spec:
       name: podinfo-pull-requests
 ```
 
+Example of inputs generated from multiple ResourceSetInputProviders via
```

Comment:
We need a sub-section in the inputs with the builtin ones providerNamespace and providerName, we also need to explain how these come from RS for inline inputs vs RSIP.

Reply:
Fixed, thanks!